src/NzbDrone.Core/Download/Clients/rTorrent/RTorrentDirectoryValidator.cs

```diff
@@ -17,8 +17,10 @@ public class RTorrentDirectoryValidator : AbstractValidator<RTorrentSettings>, I
                                           PathExistsValidator pathExistsValidator,
                                           MappedNetworkDriveValidator mappedNetworkDriveValidator)
         {
+#pragma warning disable CS0618
```

Comment:
`CascadeMode.StopOnFirstFailure` is deprecated, but still exists in FluentValidation 11:
https://docs.fluentvalidation.net/en/latest/cascade.html#stop-vs-stoponfirstfailure

I tried `CascadeMode.Stop` - it works, our tests don't crash, but I decided not to change the behavior and suppress the warning. We can simulate `StopOnFirstFailure` behavior in FluentValidation 11 using `AbstractValidator.RuleLevelCascadeMode` and `AbstractValidator.ClassLevelCascadeMode`

Reply:
If `Stop` is working then I think it makes sense to use that instead of using the deprecated `StopOnFirstFailure` and having to disable rules on it. No point making this a future us problem if we can solve it now.